lib/templates/index.html

```diff
@@ -9,7 +9,7 @@
       <dl>
         {{#package.libraries}}
           <dt>
-            <span class="name {{#isDeprecated}}deprecated{{/isDeprecated}}"><a href="{{nameForFile}}/index.html">{{ name }}</a></span>
+            <span class="name {{#isDeprecated}}deprecated{{/isDeprecated}}"><a href="{{fileName}}/{{fileName}}-library.html">{{ name }}</a></span>
```

Comment:
Should we add a getter to return the full path name? That way, we don't have to duplicate this logic in the template. We already have it in the class for the _href.